deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml

```diff
@@ -304,9 +306,25 @@ spec:
 
     The NooBaa operator deploys and manages the [NooBaa][2] Multi-Cloud Gateway on OpenShift, which provides object storage.
 
+    # Core Capabilities
+
+    * **Self-managing service:** Whichever supported storage technologies you choose, OpenShift Container Storage ensures that resources can be deployed and managed automatically.
+
+    * **Hyper-scale or hyper-converged:** With OpenShift Container Storage you can either build dedicated storage clusters or hyper-converged clusters where your apps run alongside storage.
+
+    * **File, Block, and Object provided by OpenShift Container Storage:** OpenShift Container Storage integrates Ceph with multiple storage presentations including object storage (compatible with S3), block storage, and POSIX-compliant shared file system.
+
+    * **Your data, protected:** OpenShift Container Storage efficiently distributes and replicates your data across your cluster to minimize the risk of data loss. With snapshots, cloning, and versioning, no more losing sleep over your data.
+
+    * **Elastic storage in your datacenter:** Scale is now possible in your datacenter. Get started with a few terabytes, and easily scale up. Simply add more nodes and Rook takes care of the rest.
```

Comment:
`Simply add more nodes and Rook takes care of the rest.` This is not true. Remove this sentence.